src/server/config/json/mockData.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  mockUserData: {
```

Comment:
If this object is only used in unit test, it should be inside src/test/.... directory, not here.